index.js

```diff
@@ -0,0 +1,52 @@
+var express = require("express");
+var parser  = require("body-parser");
+var hbs     = require("express-handlebars");
+var db      = require("./db/connection");
```

Comment:
nice! a little confusing because you're not using a db, but this works! consider renaming to compliments or colors